### PR TITLE
Fixed several items

### PR DESCRIPTION
**Pull request by Daegaladh**
* **Addressed Issue(s)**: N/A

* **Server Mode**: Both

* **Description of Pull Request**: Fixes several item parameters and scripts, mostly from pre-renewal

### CODE REVIEW

File: db/re/item_db.txt [hunk 1]
```diff
@@ -8336,8 +8336,8 @@
 14166,F_Giant_Fly_Wing_Box,Giant Fly Wing Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12212,10; },{},{}
 14167,F_Neuralizer_Box,Neuralizer Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12213,1; },{},{}
 14168,F_Convex_Mirror_Box,Convex Mirror Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12214,10; },{},{}
-14169,F_Blessing_10_Scroll_Box,Blessing 10 Scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12215,10; },{},{}
-14170,F_Inc_Agi_10_Scroll_Box,Increase AGI 10 scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12216,10; },{},{}
+14169,F_Blessing_10_Scroll_Box,Blessing 10 Scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12215,20; },{},{}
+14170,F_Inc_Agi_10_Scroll_Box,Increase AGI 10 scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12216,20; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt:8340]: kRO description give 10 scrolls
```suggestion
14170,F_Inc_Agi_10_Scroll_Box,Increase AGI 10 scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12216,10; },{},{}
```

File: db/re/item_db.txt [hunk 2]
```diff
@@ -8336,8 +8336,8 @@
 14166,F_Giant_Fly_Wing_Box,Giant Fly Wing Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12212,10; },{},{}
 14167,F_Neuralizer_Box,Neuralizer Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12213,1; },{},{}
 14168,F_Convex_Mirror_Box,Convex Mirror Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12214,10; },{},{}
-14169,F_Blessing_10_Scroll_Box,Blessing 10 Scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12215,10; },{},{}
-14170,F_Inc_Agi_10_Scroll_Box,Increase AGI 10 scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12216,10; },{},{}
+14169,F_Blessing_10_Scroll_Box,Blessing 10 Scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12215,20; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt:8339]: kRO description give 10 scrolls
```suggestion
14169,F_Blessing_10_Scroll_Box,Blessing 10 Scroll Box,18,20,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 12215,10; },{},{}
```

File: db/pre-re/item_db.txt [hunk 3]
```diff
@@ -6184,8 +6184,8 @@
 14166,F_Giant_Fly_Wing_Box,Giant Fly Wing Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12212,10; },{},{}
 14167,F_Neuralizer_Box,Neuralizer Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12213,1; },{},{}
 14168,F_Convex_Mirror_Box,Convex Mirror Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12214,10; },{},{}
-14169,F_Blessing_10_Scroll_Box,Blessing 10 Scroll Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12215,10; },{},{}
-14170,F_Inc_Agi_10_Scroll_Box,Increase AGI 10 scroll Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12216,10; },{},{}
+14169,F_Blessing_10_Scroll_Box,Blessing 10 Scroll Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12215,20; },{},{}
+14170,F_Inc_Agi_10_Scroll_Box,Increase AGI 10 scroll Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12216,20; },{},{}
```
Comment (Atemo) [inline db/pre-re/item_db.txt:6188]:
```suggestion
14170,F_Inc_Agi_10_Scroll_Box,Increase AGI 10 scroll Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12216,10; },{},{}
```

File: db/pre-re/item_db.txt [hunk 4]
```diff
@@ -6184,8 +6184,8 @@
 14166,F_Giant_Fly_Wing_Box,Giant Fly Wing Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12212,10; },{},{}
 14167,F_Neuralizer_Box,Neuralizer Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12213,1; },{},{}
 14168,F_Convex_Mirror_Box,Convex Mirror Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12214,10; },{},{}
-14169,F_Blessing_10_Scroll_Box,Blessing 10 Scroll Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12215,10; },{},{}
-14170,F_Inc_Agi_10_Scroll_Box,Increase AGI 10 scroll Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12216,10; },{},{}
+14169,F_Blessing_10_Scroll_Box,Blessing 10 Scroll Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12215,20; },{},{}
```
Comment (Atemo) [inline db/pre-re/item_db.txt:6187]:
```suggestion
14169,F_Blessing_10_Scroll_Box,Blessing 10 Scroll Box,18,20,,10,,,,,0xFFFFFFFF,7,2,,,,,,{ getitem 12215,10; },{},{}
```

File: db/pre-re/item_db.txt [hunk 5]
```diff
@@ -2533,7 +2533,7 @@
 5350,Pirate_Bandana_,Pirate Bandana,4,20,,100,,3,,1,0xFFFFFFFE,7,2,256,,0,1,74,{ bonus bStr,1; },{},{}
 5351,Sunflower_,Sunflower,4,20,,100,,1,,1,0xFFFFFFFF,7,2,256,,0,0,37,{ bonus2 bSubRace,RC_Insect,10; },{},{}
 5352,Poporing_Cap,Poporing Cap,4,20,,700,,2,,0,0xFFFFFFFE,7,2,256,,38,1,361,{},{},{}
-5353,Helm_Of_Sun_,Hat of the Sun God,4,20,,2400,,4,,1,0x00CFDF80,7,2,768,,0,1,138,{ bonus bStr,3; bonus bInt,2; },{},{}
+5353,Helm_Of_Sun_,Hat of the Sun God,4,20,,2400,,4,,1,0x00CFDF80,7,2,768,,0,1,138,{ bonus bStr,3; bonus bInt,2; bonus5 bAutoSpellWhenHit,"HP_ASSUMPTIO",1,1,BF_WEAPON|BF_MAGIC,0; },{},{}
```
Comment (Atemo) [inline db/pre-re/item_db.txt:2536]: Source? I can't find the HP_ASSUMPTIO part anywhere (aegis, items description)

File: db/pre-re/item_db.txt [hunk 6]
```diff
@@ -2316,7 +2316,7 @@
 5133,Sheep_Hat,Sheep Hat,4,20,,150,,1,,0,0x00008110,7,2,256,,0,0,205,{ bonus bShortWeaponDamageReturn,5; },{},{}
 5134,Pumpkin_Hat,Pumpkin-Head,4,20,,200,,2,,0,0xFFFFFFFF,7,2,256,,0,1,206,{ bonus2 bSubRace,RC_Demon,5; },{},{}
 5135,Cyclops_Visor,Cyclop's Eye,4,0,,200,,0,,0,0xFFFFFFFE,7,2,512,,75,0,207,{ bonus bMaxSP,50; },{},{}
-5136,Santa's_Hat_,Antonio's Santa Hat,4,20,,100,,3,,0,0xFFFFFFFF,7,2,256,,0,1,20,{},{},{}
+5136,Santa's_Hat_,Antonio's Santa Hat,4,20,,100,,2,,0,0xFFFFFFFF,7,2,256,,0,1,20,{},{},{}
```
Comment (Atemo) [inline db/pre-re/item_db.txt:2319]: kRO description (renewal) give 4 def, iRO give 0 def, aegis (renewal) 0
Source?

File: db/pre-re/item_db.txt (hunk 5 above)
Comment (Daegaladh) [reply db/pre-re/item_db.txt:2536]: It was confirmed long time ago that this was a hidden effect of this item, cannot find the source tho, but we already have it on our renewal db

File: db/pre-re/item_db.txt (hunk 6 above)
Comment (Daegaladh) [reply db/pre-re/item_db.txt:2319]: It's 2 according to Aegis pre-renewal
(and 4 according to my Aegis renewal, not 0 ._.)

File: db/re/item_db.txt (hunk 1 above)
Comment (Daegaladh) [reply db/re/item_db.txt:8340]: Right, I've also confirmed it on Aegis, I'll revert it back

**Pull request merged**